pom.xml

```diff
@@ -121,6 +122,27 @@
             </lifecycleMappingMetadata>
           </configuration>
         </plugin>
+          <plugin>
```

Comment:
This is the `<pluginManagement>` section. Should it instead be integrated with the existing maven-compiler-plugin definition in the main plugins section below? Otherwise it doesn't seem to run.